doc/sphinx/source/quickstart/installation.rst

```diff
@@ -214,3 +232,15 @@ confirm that no errors are reported:
 .. code-block:: bash
 
     python setup.py test
+
+.. note::
+   Using ``pip`` as installation tool is very useful since it installs in place
+   (ie inside the current source code directory, and it does not perform a copy to
+   the target install directory ``site-packages``, unlike ``python setup.py install``, which
```

Comment:
There is `python setup.py develop`, which does the same thing as `pip install -e .`. The reason we recommend `pip` is that it allows one to also install extra dependencies which are useful for development, by running `pip install -e '.[develop]'`

Reply:
heh, you know `python setup.py install` does the same in terms of installing needed deps on the fly :grin: